PharmacyManagementAPI/Services/PharmacyService.cs

```diff
@@ -1,68 +1,40 @@
-﻿using PharmacyManagementAPI.Entities;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
 using PharmacyManagementAPI.Models;
+using PharmacyManagementAPI.Repositories;
+using AutoMapper;
 
 namespace PharmacyManagementAPI.Services
 {
     public class PharmacyService : IPharmacyService
     {
         private readonly IPharmacyRepository _repository;
+        private readonly IMapper _mapper;
```

Comment:
This is not being used here. Though this is the right place for it. Move all the mapping code from repository to here.